src/Dfe.PlanTech.Application/Persistence/Extensions/IDatabaseHelperExtensions.cs

```diff
@@ -0,0 +1,60 @@
+using System.Linq.Expressions;
+using Dfe.PlanTech.Application.Persistence.Interfaces;
+using Dfe.PlanTech.Domain.Content.Models;
+
+namespace Dfe.PlanTech.Application.Persistence.Extensions;
+
+/// <summary>
+/// Various extensions methods for repeating various operations against a DbContext using the <see cref="IDatabaseHelper{TIDbContext}"/> class
+/// </summary>
+public static class IDatabaseHelperExtensions
```

Comment:
This is concrete, so without the I?
```suggestion
public static class DatabaseHelperExtensions
```

Reply:
Ah but it's extension methods for the interface (_IDatabaseHelper), not extension methods for the concrete implementation